system/perf_res_processor.py

```diff
@@ -32,6 +26,17 @@
 ]
 
 
+class PathReg:
+    base_dir = '/home/indy/indy-node/scripts/ansible/logs/'
```

Comment:
I think, that base_dir as property will be more clearly. Like:
```
class PathReg:
   def __init__(self, base_dir='/home/indy/indy-node/scripts/ansible/logs/'):
      self._base_dir=base_dir

   @property
   def base_dir():
      return self._base_dir

   @property
   def node_info_dir():
      return os.path.join(base_dir, 'node_info')

   @property
   def jctl_dir():
       return os.path.join(base_dir, 'jctl')
```

.....
etc

Reply:
Done.